pkg/virt-operator/install-strategy/create.go

```diff
@@ -3358,3 +3264,56 @@ func createOrUpdateKubeVirtCAConfigMap(
 	}
 	return nil, nil
 }
+
+func removeKvServiceAccountsFromDefaultSCC(targetNamespace string,
+	clientset kubecli.KubevirtClient,
+	stores util.Stores) error {
+
+	modified := false
+
+	SCCObj, exists, err := stores.SCCCache.GetByKey("privileged")
+	if !exists {
+		return nil
+	} else if err != nil {
```

Comment:
You should check for the error first to be on the safe side. Otherwise, if somehow `exists` will be true and `err != nil`, you will never get to the error block.

Reply:
@danielBelenky Thank you. Done.